src/templates/settings.html

```diff
@@ -317,6 +317,18 @@ <h4>Your Info:</h4>
     </div>
     <br />
     <br />
+    <div class="select center">
+        <select id="exportSelect" class="select-text" required>
+            <option>FreeTube</option>
+            <option>NewPipe</option>
+            <option>OPML</option>
+        </select>
+        <span class="select-highlight"></span>
+        <span class="select-bar"></span>
+        <label class="select-label">Subscriptions Export Format</label>
+    </div>
+    <br />
+    <br />
```

Comment:
Let's remove these 2 break lines to move the dropdown closer to the export button.  This makes it easier at a glance to know that the dropdown is associated with these buttons.